src/lib/Repository/Helper/NameSchemaService.php

```diff
@@ -6,27 +6,190 @@
  */
 namespace Ibexa\Core\Repository\Helper;
 
+use Ibexa\Contracts\Core\Persistence\Content\Type as SPIContentType;
+use Ibexa\Contracts\Core\Persistence\Content\Type\Handler as ContentTypeHandler;
 use Ibexa\Contracts\Core\Repository\Values\Content\Content;
 use Ibexa\Contracts\Core\Repository\Values\ContentType\ContentType;
+use Ibexa\Core\Base\Exceptions\InvalidArgumentType;
+use Ibexa\Core\FieldType\FieldTypeRegistry;
+use Ibexa\Core\Repository\Mapper\ContentTypeDomainMapper;
 use Ibexa\Core\Repository\NameSchema\NameSchemaService as NativeNameSchemaService;
+use Ibexa\Core\Repository\NameSchema\SchemaIdentifierExtractor;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
 
 /**
  * @deprecated inject \Ibexa\Contracts\Core\Repository\NameSchema\NameSchemaServiceInterface instead.
  * @see \Ibexa\Contracts\Core\Repository\NameSchema\NameSchemaServiceInterface
  */
 class NameSchemaService extends NativeNameSchemaService
 {
+    protected ContentTypeHandler $contentTypeHandler;
+
+    protected ContentTypeDomainMapper $contentTypeDomainMapper;
+
+    public function __construct(
+        ContentTypeHandler $contentTypeHandler,
+        ContentTypeDomainMapper $contentTypeDomainMapper,
+        FieldTypeRegistry $fieldTypeRegistry,
+        EventDispatcherInterface $eventDispatcher,
+        array $settings = []
+    ) {
+        $this->settings = $settings + [
+                'limit' => 150,
+                'sequence' => '...',
+            ];
```

Comment:
Something seems to be missing here (sequence?).
```suggestion
            'limit' => 150,
            'sequence' => '...',
        ];
```